sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -677,3 +677,10 @@ def test_categories(density, drop):
 @pytest.mark.parametrize('Encoder', [OneHotEncoder, OrdinalEncoder])
 def test_encoders_has_categorical_tags(Encoder):
     assert 'categorical' in Encoder()._get_tags()['X_types']
+
+
+def test_categorical_sort():
```

Comment:
This test should be renamed to something more explicit / specific, e.g.: `test_ordinal_encoder_deprecated_categories_auto_with_string_labels`.

Reply:
Thanks for the review ogrisel. Renamed the file.